canonical/react-components
# feat: focus SearchBox input after clearing #775

## Done

- feat: focus SearchBox input after clearing 

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Go to https://react-components-830.demos.haus/?path=/story/searchbox--external-state
- Press the clear button
- Verify the input field has focus

## Fixes

Fixes: #775
